python2/raygun4py/http_utilities.py

```diff
@@ -22,9 +22,9 @@ def build_wsgi_compliant_request(request):
         # fallback to wsgi.input
         if http_form is None and 'wsgi.input' in request:
             # we can assume WSGI keys inside this block
-            content_length = int(request.get('CONTENT_LENGTH', 0))
-            if content_length:
-                http_form = request['wsgi.input'].read(content_length)
+            content_length = request.get('CONTENT_LENGTH', 0)
+            if content_length and content_length != '':
```

Comment:
Some wsgi servers set the CONTENT_LENGTH to `''` if the client did not provide the header. This was happening in my case and the result was that the attempt to cast `''` to an `int` was raising an exception, and the `try:except` was handily swallowing the error but then a `rg_request` never got instantiated and was failing later on when trying to set `rg_request['headers'] = _headers`.